src/file/info.cr

```diff
@@ -75,8 +75,7 @@ class File
     end
   end
 
-  # A `File::Info` contains metadata regarding a file. It is returned by
-  # `File.info`, and `File#info`.
+  # A `File::Info` contains metadata regarding a file. It is returned by `File.info`.
```

Comment:
Why was this changed?

Reply:
Hmm I probably thought for some reason that the method doesn't exist.